### ISSUE TITLE
[BUG] Unable to install latest flatpak

### ISSUE THREAD

**Issue by cromelex** [aandrew-me/ytDownloader#244]
**Bug Description**
As of the last update, whenever I try to update (or clean install( ytDownloader, I get an error and can't actually install.

**To Reproduce**
Using Discover, click install. 

Always get the following error.
> Aborted due to failure (Error deploying: While trying to checkout 0326e7ef6fdff731268c875e7aca275f0d69c2f0b49d1d4914fd032ceeccd6a2 into /var/lib/flatpak/app/io.github.aandrew_me.ytdn/x86_64/stable/.0326e7ef6fdff731268c875e7aca275f0d69c2f0b49d1d4914fd032ceeccd6a2-yA58E1: Opening content object 750e2e4eb815cb641aa168a9f909b906f00e3b4f0f2f116f7619a202bae38ac0: Couldn't find file object '750e2e4eb815cb641aa168a9f909b906f00e3b4f0f2f116f7619a202bae38ac0')

**Info (please complete the following information):**
 - Operating system: Fedora 40 ( Aurora, Universal blue based)
 - App version: 3.18.2
 - Package version (flatpak/snap/other):flatpak

**Comment by aandrew-me** [comment]
Hearing such an issue for the first time. I will test when I can, did you try downloading from terminal?

**Comment by cromelex** [comment]
I just tried it via terminal and I am getting the same error.

> Error: Error deploying: While trying to checkout 0326e7ef6fdff731268c875e7aca275f0d69c2f0b49d1d4914fd032ceeccd6a2 into /var/lib/flatpak/app/io.github.aandrew_me.ytdn/x86_64/stable/.0326e7ef6fdff731268c875e7aca275f0d69c2f0b49d1d4914fd032ceeccd6a2-xzrIz1: Opening content object 750e2e4eb815cb641aa168a9f909b906f00e3b4f0f2f116f7619a202bae38ac0: Couldn't find file object '750e2e4eb815cb641aa168a9f909b906f00e3b4f0f2f116f7619a202bae38ac0'
> error: Failed to install io.github.aandrew_me.ytdn: Error deploying: While trying to checkout 0326e7ef6fdff731268c875e7aca275f0d69c2f0b49d1d4914fd032ceeccd6a2 into /var/lib/flatpak/app/io.github.aandrew_me.ytdn/x86_64/stable/.0326e7ef6fdff731268c875e7aca275f0d69c2f0b49d1d4914fd032ceeccd6a2-xzrIz1: Opening content object 750e2e4eb815cb641aa168a9f909b906f00e3b4f0f2f116f7619a202bae38ac0: Couldn't find file object '750e2e4eb815cb641aa168a9f909b906f00e3b4f0f2f116f7619a202bae38ac0'


I don't have a log of the error when I first tried to update the flatpak from previous version, but I am 99% sure I recall reading something like current version is dated/older then installed? 

It was definitely the previous release that had been available as flatpak.

**Comment by aandrew-me** [comment]
I tried installing on a fresh install of Aurora and didn't have any issues. Its likely you may have messed up something with flatpak.
Try `flatpak repair` and then `flatpak update`

**Comment by cromelex** [comment]
> I tried installing on a fresh install of Aurora and didn't have any issues. Its likely you may have messed up something with flatpak. Try `flatpak repair` and then `flatpak update`

Thanks, I will give it a try when I get a moment, and I'll report back.

**Comment by cromelex** [comment]
> I tried installing on a fresh install of Aurora and didn't have any issues. Its likely you may have messed up something with flatpak. Try `flatpak repair` and then `flatpak update`

Right, that fixed it immediately, I was able to redownload and install ytDownloader without any issue now. 

Thanks for your help.

I've only recently migrated full time from Windows and wasn't aware that flatpak repair even existed - the issue only first showed up when updating from the previous ytDownloader and was then limited to it, wasn't impacting any other flatpak I have installed. I wrongly assumed it must be related to the update! 
My bad, thanks again for your help and Keep up the good work with your app!
